lib/traject/mods_config.rb

```diff
@@ -11,15 +12,102 @@
+to_field 'cho_relation', extract_mods('/*/mods:relatedItem/mods:titleInfo[not(@*)]/mods:title')
+# Better coordinate spatial aspects?
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:cartographics/mods:coordinates')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:cartographics/mods:projection')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:cartographics/mods:scale')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:geographic')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:geographicCode')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:area')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:city')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:citySection')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:continent')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:country')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:county')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:extraterrestrialArea')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:island')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:region')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:state')
+to_field 'cho_spatial', extract_mods('/*/mods:subject/mods:hierarchicalGeographic/mods:territory')
+to_field 'cho_subject', extract_mods('/*/mods:classification')
+to_field 'cho_subject', extract_mods('/*/mods:subject/mods:topic')
+to_field 'cho_subject', extract_mods('/*/mods:subject/mods:titleInfo/mods:title')
+to_field 'cho_temporal', extract_mods('/*/mods:subject/mods:temporal')
+to_field 'cho_type', extract_mods('/*/mods:typeOfResource')
+
+# Agg
+to_field 'agg_data_provider', lambda { |_record, accumulator, context|
+  accumulator << context.settings.fetch('agg_data_provider')
+}
+to_field 'agg_is_shown_at', extract_mods('/*/mods:location/mods:url')
+to_field 'agg_provider', lambda { |_record, accumulator, context|
+  accumulator << context.settings.fetch('agg_provider')
+}
+
+# WR
+to_field 'wr_description', extract_mods('/*/mods:physicalDescription/mods:digitalOrigin')
+to_field 'wr_description', extract_mods('/*/mods:physicalDescription/mods:reformattingQuality')
+to_field 'wr_format', extract_mods('/*/mods:physicalDescription/mods:internetMediaType')
+
+# STANFORD Specific
+to_field 'wr_is_referenced_by', lambda { |_record, accumulator, context|
+  accumulator << if context.settings.fetch('identifier')
+                   inst = context.settings.fetch('inst_id') + '_'
+                   druid = context.settings.fetch('identifier').sub! inst, ''
```

Comment:
I think you want `#sub` (creates a new string) instead of `#sub!` (mutates the original) here.